### ISSUE TITLE
About the MoNuSeg dataset.

### ISSUE THREAD

**Issue by windygooo** [HUANGLIZI/LViT#26]
I've seen papers that say the MoNuSeg training set has 30 images and the test set contains 14 images? But the training data downloaded from your link has 37 images? May I ask which case do the experiments in this paper fit?
